packages/dashboard-frontend/static/registry/devfiles/empty.yaml

```diff
@@ -0,0 +1,8 @@
+schemaVersion: 2.1.0
+metadata:
+  generateName: playground
```

Comment:
let's call it empty to avoid confusion ;-)

Reply:
also, why can't we simply have it as:

```
schemaVersion: 2.2.0
metadata:
  generateName: empty
```
is it related to the https://github.com/eclipse/che/issues/21560? 

e.g. this minimal devfile would fail if we do not have the default components set in the CR e.g. 
```
  defaultComponents:
      - container:
          image: 'quay.io/devfile/universal-developer-image:ubi8-latest'
          sourceMapping: /projects
        name: universal-developer-image
```

Reply:
I have fixed it